BUILD

```diff
@@ -4,8 +4,11 @@ load("@bazel_tools//tools/python:toolchain.bzl", "py_runtime_pair")
 py_binary(
     name = "benchmark",
     srcs = ["benchmark.py"],
+    python_version = "PY2",
```

Comment:
Sigh. Please add a TODO what has to be done that we can upgrade it to Py3.

Reply:
The plan was to support this script on Python2 until its EOL right?

Reply:
I was under the impression that you added this because it only works with py2. In any case an explanatory comment would help here.

Reply:
After some checking, it's true that the script currently only works on py2 :(. In the interest of keeping the momentum and quickly producing a working version of the report, I've added Issue #36 and will address it in the future.